conf/openshift/openshift.json

```diff
@@ -526,15 +527,16 @@
       }
     },
     {
-      "kind": "DeploymentConfig",
-      "apiVersion": "v1",
+      "kind": "StatefulSet",
```

Comment:
Why is this in your pull?

Reply:
This is an unintentional change that came from a commit I cherry-picked. Removed.